src/actions/summit-event-bulk-actions.js

```diff
@@ -255,3 +291,65 @@ export const updateEventsStartDateLocal = (startDate) => (dispatch) => {
 export const updateEventsEndDateLocal = (endDate) => (dispatch) => {
     dispatch(createAction(UPDATE_END_DATE_BULK)({end_date:endDate}));
 }
+
+export const updateEventsActivityTypeLocal = (activityType) => (dispatch) => {
+    dispatch(createAction(UPDATE_ACTIVITY_TYPE_BULK)({activityType}));
+}
+export const updateEventsActivityCategoryLocal = (activityCategory) => (dispatch) => {
+    dispatch(createAction(UPDATE_ACTIVITY_CATEGORY_BULK)({activityCategory}));
+}
+export const updateEventsDurationLocal = (duration) => (dispatch) => {
+    dispatch(createAction(UPDATE_DURATION_BULK)({duration}));
+}
+export const updateEventsStreamingURLLocal = (streamingURL) => (dispatch) => {
+    dispatch(createAction(UPDATE_STREAMING_URL_BULK)({streamingURL}));
+}
+export const updateEventsStreamingTypeLocal = (streamingType) => (dispatch) => {
+    dispatch(createAction(UPDATE_STREAMING_TYPE_BULK)({streamingType}));
+}
+export const updateEventsMeetingURLLocal = (meetingURL) => (dispatch) => {
+    dispatch(createAction(UPDATE_MEETING_URL_BULK)({meetingURL}));
+}
+export const updateEventsEtherpadURLLocal = (etherpadURL) => (dispatch) => {
+    dispatch(createAction(UPDATE_ETHERPAD_URL_BULK)({etherpadURL}));
+}
+
+const normalizeEvent = (entity, eventType) => {
```

Comment:
lets refactor https://github.com/fntechgit/summit-admin/blob/ae519789d108172c81fe7ab2dbb294862914342e/src/actions/event-actions.js#L626 and use same logic on both places
knowledge should exists on one single place